.rubocop.yml

```diff
@@ -0,0 +1,67 @@
+AllCops:
```

Comment:
@digitalextremist I believe all project files should be covered by rubocop. My intention was to step-by-step shrink that list and remove excludes entirely.